ecosystem/sep-0006.md

```diff
@@ -399,7 +410,56 @@ The `fields` object contains a key for each field name and an object with the fo
 
 The wallet should display a form to the user to fill out any fields with unknown values as part of the deposit/withdrawal flow. Each field should be an text `input`, unless `choices` is specified, in which case a dropdown should be used.
 
-An anchor should also specify if they support the `/transactions` endpoint under the `"transactions"` key.
+An anchor should also indicate in the `/info` response if they support the following optional endpoints:
+
+* `transactions`
+* `transaction`
+* `fee`
+
+## Fee
+
+The fee endpoint allows an anchor to report the fee that would be charged for a given deposit or withdraw operation. This is important to allow an anchor to accurately report fees to a user even when the fee schedule is complex. If a fee can be fully expressed with the `fee_fixed` and `fee_percent` fields in the `/info` response, then an anchor should not implement this endpoint.
+
+```
+GET TRANSFER_SERVER/fee
+```
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`operation` | string | Report fee for this kind of operation (`deposit` or `withdraw`)
+`asset_code` | string | Report fee for this asset
+`amount` | float | Amount of the asset that will be deposited/withdrawn
+
+Example request:
+
+```
+GET https://api.example.com/fee?operation=withdraw&asset_code=ETH&amount=0.5
+```
+
+On success the endpoint should return `200 OK` HTTP status code and a JSON object with the following fields:
+
+Name | Type | Description
+-----|------|------------
+`fee` | float | The total fee (in units of the asset involved) that would be charged to deposit/withdraw the specified `amount` of `asset_code`
```

Comment:
Perhaps `fee`, with optional `fee_percent`, and `fee_fixed` fields so wallets could break it down for users? Just spitballing